owncloudApp/build.gradle

```diff
@@ -31,8 +31,8 @@ dependencies {
         exclude group: "com.android.support"
     }
 
-    // AppAuth
-    api 'net.openid:appauth:0.7.1'
+    // CustomTabs required for OAuth2 and OIDC.
+    implementation "androidx.browser:browser:1.3.0"
```

Comment:
Sure, It was an option. But since it is only used in this module, I added it this way.

Reply:
In my opinion, having all the versions in a file, even if any of them is being used only in one module, is easier to maintain, you don't need to remember whether that version was hardcoded in a module build.gradle or not, you will see all the versions at a glance...

I've seen this approach in other projects but you can keep it as it is since there's other hardcoded versions in the app build.gradle